tooling/cli.rs/src/dev.rs

```diff
@@ -169,6 +187,59 @@ pub fn command(options: Options) -> Result<()> {
   let (child_wait_tx, child_wait_rx) = channel();
   let child_wait_rx = Arc::new(Mutex::new(child_wait_rx));
 
+  if let AppUrl::Url(WindowUrl::External(dev_server_url)) = config
+    .lock()
+    .unwrap()
+    .as_ref()
+    .unwrap()
+    .build
+    .dev_path
+    .clone()
+  {
+    let host = dev_server_url
+      .host()
+      .unwrap_or_else(|| panic!("No host name in the URL"));
+    let port = dev_server_url
+      .port_or_known_default()
+      .unwrap_or_else(|| panic!("No port number in the URL"));
+    let addrs;
+    let addr;
+    let addrs = match host {
+      url::Host::Domain(domain) => {
+        use std::net::ToSocketAddrs;
+        addrs = (domain, port).to_socket_addrs()?;
+        addrs.as_slice()
+      }
+      url::Host::Ipv4(ip) => {
+        addr = (ip, port).into();
+        std::slice::from_ref(&addr)
+      }
+      url::Host::Ipv6(ip) => {
+        addr = (ip, port).into();
+        std::slice::from_ref(&addr)
+      }
+    };
+    let mut i = 0;
+    let sleep_interval = std::time::Duration::from_secs(5);
```

Comment:
5 seconds is too long, don't you think?

Reply:
Hmm tricky. With vite you don't even get into a second iteration, because it's too fast.
With webpack taking like 30 seconds it would get quite spammy with timeouts < 5secs.
Maybe reduce the timeout, but only log the message every x tries?

Reply:
yeah I was mainly trying to reduce the spam, since some frameworks like Quasar can take a minute to build. i'll reduce the logging instead.

Reply:
pushed, let me know what you think